docs/src/tut/wfs/index.md

```diff
@@ -0,0 +1,1343 @@
+The [Market Day](/en/essentials/getting-started/market-day/) module explores interact objects in more detail. 
+
+# {#wfs-11} Attach to the contract (buyer)
+
+This section shows you how to have the buyer attach to the contact.
+It also introduces `@reach-sh/stdlib/ask.mjs`, a minimal Reach Node.js package for command-line input.
+
+1. In `index.mjs`, add the following near the top of the file:
+
+    ``` js nonum
+    import { ask, yesno, done } from '@reach-sh/stdlib/ask.mjs';
+    ```
+
+1. And, add `done();` just before `})();` at the bottom of the file. 
+
+1. In `index.mjs`, find the following line:
+
+    ``` js nonum
+    const buyerInteract = { ...commonInteract };
+    ```
+
+1. Replace it with the following:
+
+    ``` js
+    const buyerInteract = {
+      ...commonInteract,
+      confirmPurchase: async (price) => await ask(`Do you want to purchase wisdom for ${toSU(price)} ${suStr}?`, yesno)
+    };
+
+    const acc = await stdlib.newTestAccount(iBalance);
+    const info = await ask('Paste contract info:', (s) => JSON.parse(s));
+    const ctc = acc.contract(backend, info);
+    await showBalance(acc);
+    await ctc.p.Buyer(buyerInteract);
+    await showBalance(acc);
+    ```
+
+    * Line 3: `ask` and `yesno` are functions in `@reach-sh/stdlib/ask.mjs`.
+	`yesno` accepts only `y` or `n`.
+    * Line 7: You must parse contract information (so, it must be parsable).
+    * Line 10: You can substitute `participants` for `p`.
+
+1. In `index.rsh`, find the following line:
+
+    ``` js nonum
+    const buyerInteract = { ...commonInteract };
+    ```
+
+1. Replace it with the following Reach code:
+
+    ``` js
+    const buyerInteract = {
+      ...commonInteract,
+      confirmPurchase: Fun([UInt], Bool)
+    };
+    ```
+
+    * Line 1: `buyerInteract` is a user-defined Reach object.
+    * Line 2: The spread syntax `...` adds all `commonInteract` properties (none yet) to the object.
+    * Line 3: `confirmPurchase` is a function that takes a `UInt` and returns a `Bool`.
+
+1. In `index.rsh`, add the following before `exit()`:
+
+    ``` js
+    B.only(() => { const willBuy = declassify(interact.confirmPurchase(price)); });
+    B.publish(willBuy);
+    if (!willBuy) {
+      commit();
+    } else {
+      commit();
+    }
+    ```
+
+    * Line 1: `confirmPurchase` passes `price` and returns `true` or `false` from frontend.
+    * Line 2: `B.publish()` transitions to a consensus step.
+    * Line 4: `commit()` transitions to a step.
+
+    See [Reach Mode Diagram](#reach-mode-diagram) and [Reach Mode Definitions](#reach-mode-definitions) at the bottom of this section for an explanation about `index.rsh` code.
+
+1. Run your DApp as both the seller and the buyer.
+   When prompted, copy & paste the contract info from the Seller Terminal to the Buyer Terminal.
+   Output should resemble the following:
+
+    <div class="row gx-3">
+    <div class="col-12 col-lg-auto">
```

Comment:
So... it looks like this allows things to be side-by-side.  @Craig-Elre , what would you like to write for this?  I'm not entirely certain I know what the feature should be.

I guess it could be something like:
```
::::sideBySide
:::item
imagine there is a code block here, because I don't know how to get code blocks to nest in markdown.
:::
:::item
another code block
:::
::::
```

Which... I don't think I love.  But I'm not really sure where you want this to go.  Do we want this to generalize to multiple rows/columns?  I'm happy to add an extension, but I'm not quite sure what we want.